README.md

```diff
@@ -116,16 +115,23 @@ export GITLAB_API_PRIVATE_TOKEN=<your private token from /profile/account>
 export GITLAB_API_HTTPARTY_OPTIONS="{verify: false}"
 ```
 
+Usage:
+
+When you want to know which CLI commands are supported, take a look at the client [commands implemented in this gem](http://www.rubydoc.info/gems/gitlab/3.4.0/Gitlab/Client). Any of those methods can be called as a command by passing the parameters of the commands as parameters of the CLI.
```

Comment:
You can get a list of available commands via CLI help. 

`gitlab help` lists available topics. `gitlab help Topic` will list available actions.